d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -32,15 +32,17 @@ def on_market_cycle(self, market_info):
             return
 
         for device_event in market_info['content']:
+            if 'device_info' not in device_event or device_event['device_info'] is None:
```

Comment:
```suggestion
            if not device_event.get('device_info'):
```
(Just optional, a bit more concise)